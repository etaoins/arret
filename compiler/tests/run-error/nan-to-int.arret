(import [stdlib base])
(import [stdlib test])

(defn main! () ->! ()
  (black-box! (int (black-box! ##NaN)))
  ())