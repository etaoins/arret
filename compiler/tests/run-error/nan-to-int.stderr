Float value `NaN` is not a number; cannot convert to Int
